x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/action/GetStatusAction.java

```diff
@@ -125,29 +127,39 @@ public RestStatus status() {
         }
     }
 
-    public static class Request extends AcknowledgedRequest<GetStatusAction.Request> {
-        private boolean waitForResourcesCreated;
+    public static class Request extends MasterNodeRequest<Request> {
+        private final boolean waitForResourcesCreated;
+        private final TimeValue waitForResourcesCreatedTimeout;
 
         public Request(StreamInput in) throws IOException {
             super(in);
+            waitForResourcesCreatedTimeout = in.readTimeValue();
             waitForResourcesCreated = in.readBoolean();
         }
 
-        public Request() {
-            super(TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT, DEFAULT_ACK_TIMEOUT);
```

Comment:
Fundamentally the point of this PR is to remove this usage of `TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT`, see #107984.